### ISSUE TITLE
Team update - Oct 26, 2022

### ISSUE THREAD

**Issue by trallard** [Quansight-Labs/jupyter-a11y-mgmt#165]
This is a @Quansight-Labs/czi-a11y-grant team sync 🎉🎉🎉! This is a way for the Team Members to provide status reports on what they've been up to this week and request help and attention for things they are working on. This issue will be closed at the end of the day.

_Copy and paste the template below, and answer questions as you wish!_

<details>
<summary>Response Template</summary>

```md
**Thanks I'd like to give 🙌**
- So-and-so helped me out a lot with foo...

**Updates from last week :heavy_check_mark:**
- I worked towards deliverable: <link-to-deliverable>

**Challenges I faced and things I'd like assistance with 🙏**
- I had a hard time figuring out ...

**My availability for next week**
- I'll be off on foo day...

**Important items for discussion 💬**
- I have a question about goal <link-to-goal>
- Can @foo give a comment on issue #NN ?
- I opened #NN for discussion, please chime in
- I need to discuss foo
```

</details>

#### :mag: Needs Triage

_The [Needs Triage issues](https://github.com/Quansight-Labs/jupyter-a11y-mgmt/labels/status%3A%20needs%20triage) require an initial assessment and labeling._

No issues need triage! 🎉

### :dart: Project boards

Please make sure the boards reflect the current status of the project.

[Jupyter a11y grant project](https://github.com/orgs/Quansight-Labs/projects/5/views/5)

**Comment by isabela-pf** [comment]
**Thanks I'd like to give 🙌**

**Updates from last week :heavy_check_mark:**
- #133 following up on the other app zoom review behaviors, we have the [collaborative HackMD](https://hackmd.io/0NqTDTM3TAiMYSZnof8ZWw) we used to review earlier this week. I turned our results into a table for easy comparison. 
- #132 the [keyboard navigation event has become an issue](https://github.com/jupyter/notebook/issues/6595)! Wow! That's the first attempt done. I also have notes on some feedback to address prior to the next round.
- [accessible-pygments blog post](https://github.com/quansight/quansight-website/pull/527) with Stephannie is now in PR form.
- #146 pushed more resources for test 1's [work in progress PR](https://github.com/Iota-School/notebooks-for-all/pull/26). I still need to do the synthesis work. I also made a tiny amount of progress starting the test 2 script.

**Challenges I faced and things I'd like assistance with 🙏**
- I'm surviving.

**My availability for next week**
- I'll be out on October 31 and November 4. 🎃 

**Important items for discussion 💬**
- I don't think I have anything right now. I'm still trying to get feedback on zoom for JupyterLab, but I'm not sure how we call that done.

**Comment by steff456** [comment]
**Thanks I'd like to give 🙌**
- @isabela-pf for addressing the comments of the blog post

**Updates from last week :heavy_check_mark:**
- I added two new themes for the `accessible-pygments` package - https://github.com/Quansight-Labs/accessible-pygments/pull/14
- Started investigating the status of [jupyterlab accessible themes](https://github.com/Quansight-Labs/jupyterlab-accessible-themes) repo:
    - The repo right now is very similar to the output of the jupyterlab's cookie cutter
    - The contents cannot be installable, so I'm not sure what's inside of it - all the readme instructions don't work
    - For future tasks inside the repo, I think we need to do a big cleanup and fix the repo
- Realized that `pygments` isn't used as a backend for syntax highlighting in Jupyterlab
    - Right now jupyterlab uses `marked.js` which defaults to `highlight.js`, so we are not going to be able to use the accessible pygments extension
    - There's some issues regarding how to use pygments for the **output** of the cells

**Challenges I faced and things I'd like assistance with 🙏**
- Defining what to do with `accessible-pygments` and next steps for color issues.

**My availability for next week**
- I'll be here

**Important items for discussion 💬**
- Next steps for the theme issue,
    - Do we want Jupyter themes with the colors of the themes available in `accessible-pygments`?

**Comment by gabalafou** [comment]
**Thanks I'd like to give 🙌**
- @kulsoomzahra for putting in a lot of effort on the blog post!
- @isabela-pf for helping me on the internship blog post
 
**Updates from last week :heavy_check_mark:**
- Helped Kulsoom finish blog post
- https://github.com/Quansight-Labs/jupyter-a11y-testing/pull/25 - created PR of my testing work from the past two weeks
- Working on fixing keyboard navigation issues in JupyterLab. Right now, I'm focussing on focus indicators.

**Challenges I faced and things I'd like assistance with 🙏**
- I had a hard time figuring out what to do with the internship blog post, but @isabela-pf helped me out.
- I need to do contrast ratio checking in the Playwright tests but I'm not sure what library or algorithm to use. 

**My availability for next week**
- Here all week!

**Important items for discussion 💬**
- I'm not sure how I should stage and sequence my work on keyboard navigability. There are a lot of small changes I want to make. Each small change should be its own PR, but they should probably all ship together to avoid UX churn. For example, one of the changes I want to make is to put a visible focus indicator around each UI element that can receive focus. This would result in visible focus rings around the file names in the file browser. However, in another PR, I want to remove the files from the tab index. I don't want each individual file to be navigable by the tab key; rather, the user should tab into the file list and then use the up/down arrow keys to select individual files.
- I have questions about color contrast and the [new WCAG 2.2 guidelines for the focus indicator](https://www.w3.org/WAI/WCAG22/Understanding/focus-appearance-minimum)

**Comment by isabela-pf** [comment]
Follow up notes:

- Stephannie: has ideas of how to move forward with accessible-pygments, but needs to see if they are aligned with Tania's plans. They have time scheduled for this.
- Gabriel asking for some review on https://github.com/Quansight-Labs/jupyter-a11y-testing/pull/25
     - Isabela to review contents of `test-info-pages`
     - Stephannie to review anything!
